src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemProviderFactory.cs

```diff
@@ -15,15 +15,14 @@ internal class NavigateToItemProviderFactory : INavigateToItemProviderFactory
         private readonly IAsynchronousOperationListener _asyncListener;
 
         [ImportingConstructor]
-        public NavigateToItemProviderFactory(
-            [ImportMany] IEnumerable<Lazy<IAsynchronousOperationListener, FeatureMetadata>> asyncListeners)
+        public NavigateToItemProviderFactory(IAsynchronousOperationListenerProvider listenerProvider)
```

Comment:
@minestarks this basically shows how you need to change code.

importing IEnumerable<IAsync, FeatureMetadata> listeners and then creating new AggregateAsync..Operation pattern goes away and changed to

import IAsync..OperationListenerProvider and calling listenerProvider.GetListener(...)

Reply:
@minestarks type/members showing up here - 5d7b8ba4db21a7264654a7d30013f2e9b0cea1bc - will go away soon. for now I didn't put [Obsolete] but I will do that in next insertion so that we can remove those soon.